libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html

```diff
@@ -76,3 +81,37 @@ <h1 class="header-grid-area-name">
     </div>
   </ng-container>
 </ng-container>
+
+<ng-container
+  *transloco="let t; read: 'marketParticipant.gridAreas.changeGridArea'"
+>
+  <ng-container *ngIf="editGridNameEditOpen">
+    <!--<watt-modal
+      #nameChangeModal
+      [title]="'Rediger netområde'"
+      [size]="'small'"
+      [disableClose]="false"
+      closeLabel="Close modal">
+    <br />-->
+    <watt-form-field>
+      <watt-label>{{ t("gridAreaName") }}</watt-label>
+      <input
+        #nameChangeInput
+        wattInput
+        [(ngModel)]="newGridName"
+        type="text"
+        required
+        [formControl]="nameChangeForm"
```

Comment:
suggestion: the form input is using both types of forms in Angular (`[(ngModel)]` and `[formControl]`). Let's use only one. The `nameChangeForm` property is not used anywhere else in the template and the class, so let's remove it. Also the related `ReactiveFormsModule.`